### PR TITLE
xa65, new recipe

### PR DESCRIPTION
**Pull request by Begasus**
Needed to checkout latest vice release ...

### CODE REVIEW

File: dev-lang/xa65/xa65-2.3.8.recipe [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+DESCRIPTION="xa is a high-speed, two-pass portable cross-assembler. It \
+understands mnemonics and generates code for NMOS 6502s (such as 6502A, \
+6504, 6507, 6510, 7501, 8500, 8501, 8502 ...), CMOS 6502s (65C02 and \
+Rockwell R65C02) and the 65816."
+HOMEPAGE="https://www.floodgap.com/retrotech/xa/"
+COPYRIGHT="1989-2017 Andre Fachat
+	Jolse Maginnis
+	David Weinehall
+	Cameron Kaiser"
+LICENSE="GNU LGPL v2"
```
Comment (alaviss) [inline dev-lang/xa65/xa65-2.3.8.recipe]: It's [the GPL, not LGPL][0]

```diff
-LICENSE="GNU LGPL v2"
+LICENSE="GNU GPL v2"
```

[0]: https://www.floodgap.com/retrotech/xa/COPYING.txt

**Comment by pulkomandy** [comment]
Unfortunately, using env and extra arguments to perl in the shebang creates problems on some POSIX systems (hi Linux!) It will call env with the single argument "perl -s", and this won't work.

File: dev-lang/xa65/patches/xa65-2.3.8.patchset [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+Date: Tue, 26 Jun 2018 16:01:29 +0200
+Subject: use /bin/perl instead of /usr/bin/perl
+
+
+diff --git a/tests/harness b/tests/harness
+index fdc7d53..da641a4 100755
+--- a/tests/harness
++++ b/tests/harness
+@@ -1,4 +1,4 @@
+-#!/usr/bin/perl -s
```
[the remark this answers is not in the document]
Comment (Begasus) [reply dev-lang/xa65/patches/xa65-2.3.8.patchset]: I'll leave this for now, until there is a clear answer, I've used the 'env' settings in the past, waiting for @pulkomandy his response on this matter :)

File: dev-lang/xa65/xa65-2.3.8.recipe (hunk 1 above)
Comment (Begasus) [reply dev-lang/xa65/xa65-2.3.8.recipe]: Will fix :)

**Comment by Begasus** [comment]
@alaviss I wasn't sure about the naming xa or xa65 as I can't find any instance of it on gentoo ...

**Comment by alaviss** [comment]
@Begasus I think it's fine either way :)

**Pull request merged**